lua/autorun/client/stylesheet.lua

```diff
@@ -112,4 +288,159 @@ function draw.GradientShadowUp(x, y, w, h, alpha)
     if cornerMat:IsError() then return end
     predrawshadow(alpha)
     surface.DrawTexturedRectUV(x, y, w, h, 1, 0, 1, 1)
-end
\ No newline at end of file
+end
+
+surface.CreateFont('SS_Heading', {
```

Comment:
Can you put swampshop-specific styles in a different file? lua/swampshop/stylesheet.lua (and make sure to include it)

Reply:
i kinda figured most of this would probably be useful to reuse elsewhere